spa/src/main.ts

```diff
@@ -1,17 +1,18 @@
-import Vue from "vue"
+import Vue from "vue";
 import VueRouter from "vue-router";
 import VueGtag from "vue-gtag";
 
-import App from "./App.vue"
+import App from "./App.vue";
 import api from "./api";
 import appStore from "./appStore";
 import { User } from "./appStore";
-import * as filters from './helpers/fiters';
+import * as filters from "./helpers/fiters";
 import routes from "./routes";
 import socket from "./socket";
 import "./assets/index.scss";
+import {name} from "dayjs";
```

Comment:
Why is name from dayjs being imported here?

Reply:
Because my IDE doesn't mind it's business hahaha